back-end/helpers/test-helpers/testIntentGenerator.js

```diff
@@ -0,0 +1,29 @@
+export default class TestIntentGenerator {
+    generateModel(questionMsg, intents, total_intents, id) {
+        const generatedModel = {
+            question: questionMsg,
+            children: intents,
+            total_children: total_intents,
+            project_id: id,
+            getPercentages: jest.fn()
+        };
+        return generatedModel;
+    };
+
+    generateAnalyzedIntent(questionMsg, intents) {
+        const generatedIntent = {
+            question: questionMsg,
+            children: intents,
+        };
+
+        return generatedIntent;
+    }
+
+    generateMultipleIntents(intentInputs) {
+        const intents = []
+
+        intentInputs.forEach((input) => {intents.push(this.generateAnalyzedIntent(input[0], input[1]))})
+
+        return intents;
+    }
+}
```

Comment:
nit: add the istanbul thing so this is ignored in test coverage. Also renaming the file/class to Factory might be nice

Reply:
uhhh what is the istanbul thing? >.>

Reply:
```
/* istanbul ignore file */
```
At the top of the file pls